pyomo/solvers/tests/checks/test_CPLEXDirect.py

```diff
@@ -262,7 +263,27 @@ def test_soln_limit_mip(self):
                              TerminationCondition.unknown)
             self.assertEqual(model.solutions[0].status,
                              SolutionStatus.stoppedByLimit)
-            self.assertGreater(results.solver.deterministic_time, 0.0)
+
+    def test_dettime_limit_mip(self):
```

Comment:
@begunefeoglu I think we need to wrap this in the same `@unittest.skipIf` decorator otherwise the pipeline will fail if CPLEX is not available